src/main/java/game/Game.java

```diff
@@ -35,6 +35,10 @@
 	 */
 	private final Spawner spawner;
 	/**
+	 * Handles audio for game.
+	 */
+	private final Audio audio;
+	/**
```

Comment:
we maken hier later wel een singleton van, daar hoef je nu nog niets aan te doen, behalve als je weet wat ik bedoel (net zoals de logger)